plugins/quay/README.md

```diff
@@ -52,7 +52,9 @@ This plugin will show you information about your container images within Quay re
 
 ## Development
 
-In [Backstage plugin terminology](https://backstage.io/docs/local-dev/cli-build-system#package-roles), this is a `frontend-plugin`. However it requires backend proxy to be available at all times. Development environment therefore requires you to run a backend instance as well. You can start a live dev session from the repository root using following commands concurrently:
+In [Backstage plugin terminology](https://backstage.io/docs/local-dev/cli-build-system#package-roles), the Quay plugin is a frontend plugin. However, it requires backend proxy to be available at all times. Therefore, you need to run a backend instance in the development environment as well.
+
+You can run the following commands concurrently to start a live development session from the repository root:
```

Comment:
```suggestion
You can run the following commands concurrently from the repository root to start a live development session:
```

Just a nit. I think this makes more sense since you're running the *commands* from the repository root, not the *live development session*. WDYT?

Reply:
Sounds good to me.